src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/LoginPage.java

```diff
@@ -31,27 +36,73 @@
 
     @FindBy(id = "login_button")
     private ExtendedWebElement loginBtn;
+    @FindBy(id = "login_button")
+    private ExtendedWebElement signUpButton;
 
     public LoginPage(WebDriver driver) {
         super(driver);
     }
 
+
     @Override
     public void typeName(String name) {
         nameInputField.type(name);
         hideKeyboard();
     }
 
+
+    public String getTypeName() {
+        return nameInputField.getText();
+    }
+
+    public String getTypePassword() {
+        return passwordInputField.getText();
+    }
+
+
+    @Override
+    public boolean verifyFieldsArePresented() {
+        return nameInputField.isElementPresent() &&
+                passwordInputField.isElementPresent() &&
+                maleRadioBtn.isElementPresent() &&
+                femaleRadioBtn.isElementPresent() &&
+                privacyPolicyCheckbox.isElementPresent();
+    }
```

Comment:
bad practice
if you need verify a lot of fields use one by one(one method for one field)
Example: isNameFieldPresented, ....